tests/hostcfgd/sample_output/SSH_SERVER_modify_all/sshd_config

```diff
@@ -53,7 +53,7 @@ MaxAuthTries 16
 #IgnoreRhosts yes
 
 # To disable tunneled clear text passwords, change to no here!
-PasswordAuthentication yes
```

Comment:
>PasswordAuthentication [](http://example.com/codeflow?start=0&length=22)

Is this new feature backward-compatible? If the new config is not used, the default value should ensure sshd_config same as old behavior, so it does not break existing production usage easily. We can keep this file unchanged to test backward-compatible. If you need, you can add extra testcases (test output files).